recipes/traits/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: traits
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/enthought/traits
+  git_tag: {{ version }}
+
+build:
+  number: 0
```

Comment:
Add `script: python setup.py install` here and remove `build.sh` and `bld.bat`.

Does this use `setuptools`?